src/components/modals/SettingsModal.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+import { Modal, Button } from 'react-bootstrap';
+import FaShareAlt from 'react-icons/lib/fa/share-alt';
+import FaExternalLink from 'react-icons/lib/fa/external-link';
+
+import { chatHandler } from '../../epics/chatEpics';
+
+const shareLink = window.location.href;
+
+const onDeleteAllMsgs = (e) => {
+  if (window.confirm("Are you sure you want to delete every existing chat message from this chat room? This action cannot be undone.")) {
+    chatHandler.sendDeleteAllMessagesSignalToServer();
+  }
+}
+
+const onCopyShareLink = (e) => {
+  navigator.clipboard.writeText(shareLink);
+}
+
+const SettingsModal = ({
+  isVisible,
+  onCloseModal
+}) => {
+
+  return (
+    <div>
+      <Modal show={isVisible} onHide={onCloseModal}>
+        <Modal.Header closeButton>
+          <h2>Settings</h2>
+        </Modal.Header>
+        <Modal.Body>
+          <hr />
+          <h3>Invite to this Chat</h3>
+          <p>
+            Click to copy a link and invite others to this LeapChat room.
+          </p>
+          <Button onClick={onCopyShareLink} bsStyle="primary">
+            Copy Invite Link <FaShareAlt />
+          </Button>
+          <hr />
+          <h3>Feedback</h3>
+          <p>
+            Do you have feedback or suggestions on how we can improve LeapChat? We're listening!{' '}
+            <a href="https://github.com/cryptag/leapchat/issues/new" target="_blank">
```

Comment:
I just realized (and tested): if they don't have a GitHub account, they'll get a redirect to a login page. Maybe point them to github.com/cryptag/leapchat/issues where they can see other people's feedback and probably notice the "New Issue" button in the upper right? Thankfully that button shows up even if the user isn't logged in, and it encourages the user to create a GitHub account in order to create an issue.

Thoughts?

Reply:
Ah, bummer. That sounds good.